docs/modules/ROOT/pages/advanced-how-tos/managing-security-fix.adoc

```diff
@@ -1 +1,100 @@
-= Managing a security fix
\ No newline at end of file
+= Managing a security fix
+
+Sometimes, you need to address an **embargoed security issue** in your application. So as not to disclose aspects of the security vulnerability prematurely, you will want to prepare a fix in private or with a limited set of collaborators. This document guides you on how to make a private copy of your Git repository and clone your application to a private workspace where you can prepare a fix and invite other collaborators only on a need-to-know basis, so that you can control who is aware of the issue while you develop the fix.
+
+Key steps include:
+
+* **Clone affected application:** Cloning affected applications into a private workspace, enabling developers to tackle security issues without exposing sensitive information.
+* **Prepare the fix:** Preparing the fix with source code in your private git mirror and builds in your private workspace.
+* **Release the fix:** Releasing the build from your private workspace and "upstreaming" your patch from your private git mirror back to your team's normal development repo.
+
+== Cloning applications to a private workspace
+Cloning an application involves replicating the application's code and configuration into a dedicated private workspace, ensuring that sensitive information remains isolated and protected.
+
+.**Prerequisites**
+
+* You have the Maintainer role or greater in the original workspace where the embargoed issue was identified.
+// Maintainer role was xref:getting-started/roles_permissions/ Currently no such file in konflux-ci/docs. Add a link if the doc is published.
+* You have link:https://konflux-ci.dev/docs/getting-started/cli/[enabled CLI access for {ProductName}].
```

Comment:
The line used to say "You have downloaded the RHTAP CLI". Do we expect Konflux CLI in the future?
In other words, is it safe to link to the Getting started page or should I leave a comment to provide the new CLI link later on?

Reply:
@gbenhaim @yftacherzog , is there a plan to have a `konflux` CLI to replace the `rhtap` one? This document references a `rhtap` CLI... 

@missmesss, I think that the CLI page is currently scoped to configuring your local environment to interact with the Konflux deployment via `kubectl` or `oc` and not the "platform" CLI. I am not sure if the link is relevant as we have it as this dependency is referenced on like 37 (but that _can_ be moved up here as well).

Reply:
I will have to defer to @gbenhaim and @yftacherzog . It likely still needs to be here but there is still the open question about the `konflux` CLI.

Reply:
Sounds to me as this requires onboarding a new application that points to the private repo.

@gbenhaim any thoughts?